src/Sarif/Errors.cs

```diff
@@ -201,6 +202,26 @@ public static void LogExceptionAccessingFile(IAnalysisContext context, string fi
             context.RuntimeErrors |= RuntimeConditions.ExceptionAccessingFile;
         }
 
+        public static void LogInvalidInvocationPropertyName(IAnalysisContext context, string propertyName)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            // '{0}' is not a property of the Invocation object.
+            context.Logger.LogConfigurationNotification(
+                CreateNotification(
+                    context.TargetUri,
+                    ERR997_InvalidInvocationPropertyName,
+                    NotificationLevel.Error,
+                    null,
```

Comment:
provide name of parameter here for the embedded literals (and all subsequent args, of course, which is required)